llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -120,6 +122,8 @@ bool VPRecipeBase::mayReadFromMemory() const {
     return !cast<VPWidenCallRecipe>(this)
                 ->getCalledScalarFunction()
                 ->onlyWritesMemory();
+  case VPWidenIntrinsicSC:
+    return true;
```

Comment:
Same

Reply:
Yep, updated to store read/write/sideeffcts in recipe.